src/main/java/hng_java_boilerplate/product/errorhandler/ProductErrorHandler.java

```diff
@@ -37,4 +39,24 @@ public ResponseEntity<?> handleException(NoHandlerFoundException ex){
         productErrorDTO.setSuccess(false);
         return new ResponseEntity<>(productErrorDTO, HttpStatus.NOT_FOUND);
     }
+
+    @ExceptionHandler
+    public ResponseEntity<?> handleException(ProductNotFoundException ex){
+        productErrorDTO.setMessage(ex.getMessage());
+        productErrorDTO.setStatus_code(HttpStatus.NOT_FOUND.value());
+        productErrorDTO.setSuccess(false);
+        return new ResponseEntity<>(productErrorDTO, HttpStatus.NOT_FOUND);
+    }
+
+    @ExceptionHandler(MethodArgumentNotValidException.class)
+    public ResponseEntity<?> handleValidationException(MethodArgumentNotValidException ex) {
```

Comment:
An exception handler that handles methodargumentnotvalid already exists in the applicaition

Reply:
Please, I did not see that method in "ProductErrorHandler"